<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestGraph</class>
 <widget class="QMainWindow" name="TestGraph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test Graph</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Test Graph</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="gview" native="true">
         <layout class="QVBoxLayout" name="gviewLayout"/>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="buttonsLayout">
         <item>
          <widget class="QPushButton" name="button_add">
           <property name="minimumSize">
            <size>
             <width>95</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
           <property name="class" stdset="0">
            <string>Console</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_remove">
           <property name="minimumSize">
            <size>
             <width>95</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
           <property name="class" stdset="0">
            <string>Console</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_refresh">
           <property name="minimumSize">
            <size>
             <width>95</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Refresh</string>
           </property>
           <property name="class" stdset="0">
            <string>Console</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="vspacer1">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
